### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by robgpita**
Change example of verifying files are in NetCDF Data Format in Prerequisites section to reflect the yppm, not c_sw. These examples/output could change if some of these files are deleted with the outstanding feature/interpolate PR.

### CODE REVIEW

**Review by christopherwharrop-noaa — APPROVED** [review]

File: ref/README.md
Comment (LyndStringer) [inline ref/README.md:41]: I thought these old files were gone.  I do not understand.  Why put them back?  In the README.md?

**Review by LyndStringer — CHANGES_REQUESTED** [review]

File: ref/README.md
Comment (christopherwharrop-noaa) [reply ref/README.md:41]: They are not gone.  This fixes a cut-and-paste error that I made when updating the README file.  I had copied it from the c_sw, and so the filenames were incorrect.  These files are not gone, they are still there.  PR #11 proposes to remove some of those files, but it hasn't happened yet.  This change is consistent with `develop`.

**Comment by christopherwharrop-noaa** [comment]
@LyndStringer - Can you please re-review this PR so we can get it merged?  I believe your comment was addressed, but since you requested changes, I can't merge until you approve.

**Review by LyndStringer — APPROVED** [review]

**Pull request merged**
